src/guides/v2.3/rest/notes.md

```diff
@@ -1,22 +1,22 @@
 ---
 group: rest-api
-subgroup: A_rest
 title: REST usage notes
-menu_title: REST usage notes
-menu_order: 7
 functional_areas:
   - Integration
 ---
 
-## Parameters in URLs
+This topic describes issues you might encounter when using REST endpoints.
```

Comment:
Why say "this topic"? I tend to treat content as if it could fit anywhere, even like an include; never know in the future. Why not something like "You might encounter the following issues when using REST endpoints."

Also, this topic is so short. Are you expecting more issues to mention?

Reply:
This topic exists because there was (and continues to be) no other place to put this info. But yes, at one point I expected this page to contain more content.